### ISSUE TITLE
Rework Content Tab

### ISSUE THREAD

**Issue by Fixer-007** [minetest/minetest#8032]
##### Issue type

 - Feature request

##### Minetest version

```
5.0+
```

##### Summary
Minetest has "Content Tab", that has list of installed packages with ability to uninstall them.
After introduction of "Browse Online Content" button, you notice pretty much the same functionality hidden in it, with download/list/uninstall package abilities.

It is obvious, "Content Tab" is redundant, you should dump it entirely and replace with "Browse Online Content" UI with slight visual modifications to fit it nicer, the only addition to it is listing of externally installed package with mark "external" and ability to remove those.

@rubenwardy said there will be some challenges to fit it in tab system, but I still think it is good way.

Content DB should be "right in your face" in the tab itself, not hidden in some far buttons few will notice. Embrace it.

**Comment by Wuzzy2** [comment]
> It is obvious, "Content Tab" is redundant, you should dump it entirely and replace with "Browse Online Content" UI with slight visual modifications to fit it nicer, the only addition to it is listing of externally installed package with mark "external" and ability to remove those.

I disagree because you might obtain a mod from places other than the Content DB. Therefore, users will lose the ability to uninstall those mods. Your suggestion will be a downgrade.

**Comment by Fixer-007** [comment]
>I disagree because you might obtain a mod from places other than the Content DB. Therefore, users will lose the ability to uninstall those mods. 

If you reread my post, it says CDB will handle externally installed mods as well, with ability yo uninstall them.

**Comment by Wuzzy2** [comment]
I still don't like it, it doesn't seem like ideal usability. What makes you think that Average Joe will understand what “external” means?

Please draw a mock-up so we have an idea what you actually want. Otherwise, I think it is futile to discuss this further.

**Comment by rubenwardy** [comment]
I'm against this due to available size and logistical reasons
